Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -51,7 +51,7 @@ If any setting is unspecified, a suitable default will be used automatically.
 - `blockPoolName`: The metadata name of the CephBlockPool CR where the rados namespace will be created.
 
 - `mirroring`: Sets up mirroring of the rados namespace (requires Ceph v20 or newer)
-    - `mode`: mirroring mode to run, possible values are "pool" or "image" (required). Refer to the [mirroring modes Ceph documentation](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#enable-mirroring) for more details
+    - `mode`: mirroring mode to run, possible "pool", "image" or "init-only" (required). Refer to the [mirroring modes Ceph documentation](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#enable-mirroring) for more details
```

Comment:
radosnamespace will not have init-only mode
https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#namespace-configuration

Could we update the link for radosnamespace mirroring as well?

Reply:
done